cli/azd/pkg/project/scaffold_gen.go

```diff
@@ -165,6 +165,23 @@ func infraSpec(projectConfig *ProjectConfig) (*scaffold.InfraSpec, error) {
 			}
 
 			infraSpec.Services = append(infraSpec.Services, svcSpec)
+		case ResourceTypeOpenAiModel:
```

Comment:
Feels like a deja-vu to Aspire origins.  There should be learnings about why the strategy of manually adding supported resources was not solid and made us pivot from it.

Reply:
We will be rewriting the backend -- it will scale; just not for alpha  👌